traffic_server/_tsb/trafficserver.spec

```diff
@@ -20,9 +20,9 @@
 
 %global src %{_topdir}/SOURCES/src
 %global git_args --git-dir="%{src}/.git" --work-tree="%{src}"
-%global tag %(git %{git_args} describe --long --tags --match='*[0-9.][0-9.][0-9.]' |      sed 's/^\\\(.*\\\)-\\\([0-9]\\\+\\\)-g\\\([0-9a-f]\\\+\\\)$/\\\1/' | sed 's/-/_/')
-%global distance %(git %{git_args} describe --long --tags --match='*[0-9.][0-9.][0-9.]' | sed 's/^\\\(.*\\\)-\\\([0-9]\\\+\\\)-g\\\([0-9a-f]\\\+\\\)$/\\\2/')
-%global commit %(git %{git_args} describe --long --tags --match='*[0-9.][0-9.][0-9.]' |   sed 's/^\\\(.*\\\)-\\\([0-9]\\\+\\\)-g\\\([0-9a-f]\\\+\\\)$/\\\3/')
+%global tag %(git %{git_args} describe --long --tags --match='*[0-9.][0-9.][0-9.]' --match='*[0-9.][0-9.][0-9.]-rc[0-9]'      | sed 's/^\\\(.*\\\)-\\\([0-9]\\\+\\\)-g\\\([0-9a-f]\\\+\\\)$/\\\1/'  | sed 's/-/_/g')
+%global distance %(git %{git_args} describe --long --tags --match='*[0-9.][0-9.][0-9.]' --match='*[0-9.][0-9.][0-9.]-rc[0-9]' | sed 's/^\\\(.*\\\)-\\\([0-9]\\\+\\\)-g\\\([0-9a-f]\\\+\\\)$/\\\2/')
+%global commit %(git %{git_args} describe --long --tags --match='*[0-9.][0-9.][0-9.]' --match='*[0-9.][0-9.][0-9.]-rc[0-9]'   | sed 's/^\\\(.*\\\)-\\\([0-9]\\\+\\\)-g\\\([0-9a-f]\\\+\\\)$/\\\3/')
```

Comment:
The `9.2.4` tag has been pushed to the ATS repo now, so the `-rc` references can be removed. ATS votes are usually very quick, so release candidates usually don't stick around as the latest tag for very long.

Reply:
maybe the pushed it after I reported to them on traffic-server channel ;))

Reply:
reverted it